src/components/Dropdown/Dropdown.driver.ts

```diff
@@ -73,6 +85,84 @@ export const dropdownDriverFactory = (base: UniDriver): DropdownDriver => {
       tooltipDriver.mouseEnter();
       return tooltipDriver.getContentElement().innerHTML;
     },
+  };
+};
+
+const nativeDriver = (
+  base: UniDriver,
+  baseUniDriver: BaseUniDriver,
+): BaseDropdownDriver => {
+  const getNativeOptions = () =>
+    base.$$(`option:not([data-hook="${DATA_HOOKS.placeholderOption}"])`);
+  const getNativeSelectElement = async () =>
+    base.$(`[data-hook="${DATA_HOOKS.nativeSelect}"]`).getNative();
+  const warnUnsupportedFunction = (fnName: string) => {
+    console.warn(
+      `Function ${fnName} is not supported within native dropdown mode.`,
+    );
+    return null;
+  };
+
+  return {
+    isDisabled: async () =>
+      (await (await getDropdownNativeSelect(base)).attr('disabled')) !== null,
+    getAriaLabel: async () =>
+      (await getDropdownNativeSelect(base)).attr('aria-label'),
+    getAriaLabelledBy: async () =>
+      (await getDropdownNativeSelect(base)).attr('aria-labelledby'),
+    hasAriaHasPopup: async () => warnUnsupportedFunction('hasAriaHasPopup'),
+    click: async () => warnUnsupportedFunction('click'),
+    areOptionsShown: async () => warnUnsupportedFunction('areOptionsShown'),
+    selectOptionAt: async (index: number) => {
+      const option = getNativeOptions().get(index);
+      const isDisabled = (await option.attr('disabled')) !== null;
+      !isDisabled && Simulate.change(await option.getNative());
+    },
+    getOptionsCount: async () => getNativeOptions().count(),
+    hasErrorMessage: async () =>
+      (await getTooltipDriver(baseUniDriver)).exists(),
+    hasError: async () =>
+      (await (await getDropdownNativeSelect(base)).attr(
+        DATA_ATTRIBUTES.error,
+      )) !== null,
+    getErrorMessageContent: async () => {
+      const tooltipDriver = await getTooltipDriver(baseUniDriver);
+      tooltipDriver.mouseEnter();
+      return tooltipDriver.getContentElement().innerHTML;
+    },
+    getDropdownText: async () =>
+      (await getNativeSelectElement()).selectedOptions[0].text,
```

Comment:
hmmm, I think this means this won't work for e2e tests.
When you called `native()` up there ☝️ the unidriver returns the environment driver.
In our tests (mocha) the native driver can fetch the `selectedOptions`, because it has access to the dom (jsdom in this case).
But in e2e tests it won't work.

I think we need to use Unidriver's `_prop` method instead:
https://github.com/wix-incubator/unidriver/blob/20c9935d2be9ca3caf717ab6235d1a4a0346e467/core/src/index.ts#L47

Reply:
If i'm wrong, and you'ld like to prove it, you can add an `e2e` test file, testing this new method.
If it works we're all good

Reply:
It may cause problems with IE 11 e2e tests because it is not supported (and probably not polyfilled) in IE11. I'll try using _prop method